stock_available_unreserved/models/stock_quant.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 Camptocamp SA
+# Copyright 2016-19 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+
+
+class StockQuant(models.Model):
+    _inherit = "stock.quant"
+
+    contains_unreserved = fields.Boolean(
```

Comment:
@sbejaoui Is this field still needed as there is a field 'available_quantity'? The initial compute is quite slow.

https://github.com/odoo/odoo/commit/4afba9ee68918882259bd26bd07003bce0ddb3a3